.github/workflows/docker-build-push.yml

```diff
@@ -41,3 +41,13 @@ jobs:
         run: |
           docker build --build-arg SVC="${{ matrix.service_name }}" -t $GHCR/"${{ matrix.service_name }}":${{ github.sha }} -f "${{ matrix.service_name }}"/Dockerfile .
           docker push $GHCR/"${{ matrix.service_name }}":${{ github.sha }}
+  test:
```

Comment:
@marrobi better name for this, instead of test.

Reply:
@shajidissa suggest one, I thought "build" followed by "test" was ok!